src/web/mjs/connectors/FrScan.mjs

```diff
@@ -0,0 +1,31 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class FrScan extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'fr-scan';
+        super.label = 'FR-Scan';
+        this.tags = [ 'manga', 'webtoon', 'french' ];
+        this.url = 'https://fr-scan.com/';
+        this.requestOptions.headers.set('x-referer', this.url);
+        this.language = 'fr';
+
+        this.queryTitleForURI = 'div.profile-manga div.post-title h1';
+    }
+
+    get icon(){
+        return '/img/connectors/fr-scan';
+    }
+
+    async _getPages(chapter) {
+        const request = new Request(new URL(`${chapter.id}`, this.url), this.requestOptions);
+        const data = await this.fetchDOM(request, 'div.page-break');
+        return data.map(element => {
+            let srcUrl = element.innerHTML;
```

Comment:
any reason for this? 

If the reason is to get the image in best quality, madara Template is taking care of that.

https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/connectors/templates/WordPressMadara.mjs#L146

Reply:
super function doesn't retrieve image (null), this is the only I achieved to get Image. Probably not the best one :) I'll try on my side with your input

Reply:
I see. 

![image](https://github.com/manga-download/hakuneko/assets/10671518/8a1f3232-abff-45fa-8619-4a113196a181)

Suggestion : 

* Use a proper CSS selector for images that allows you to get the images inside the noscript tag only, because their src attribute got the good url. You just have to override this.queryPages.

or 

* override _getpages, get only pages with [data-lazy-src] attribute, and return that attribute value.